src/databricks/labs/ucx/hive_metastore/table_migration_status.py

```diff
@@ -153,10 +155,19 @@ def _try_fetch(self) -> Iterable[TableMigrationStatus]:
         for row in self._fetch(f"SELECT * FROM {escape_sql_identifier(self.full_name)}"):
             yield TableMigrationStatus(*row)
 
+    def _iter_catalogs(self) -> Iterable[CatalogInfo]:
+        try:
+            yield from self._ws.catalogs.list()
+        except DatabricksError as e:
+            logger.warning("Cannot list catalogs", exc_info=e)
+
     def _iter_schemas(self):
-        for catalog in self._ws.catalogs.list():
+        for catalog in self._iter_catalogs():
             try:
                 yield from self._ws.schemas.list(catalog_name=catalog.name)
             except NotFound:
                 logger.warning(f"Catalog {catalog.name} no longer exists. Skipping checking its migration status.")
                 continue
+            except DatabricksError as e:
+                logger.warning(f"Received error while listing schemas in catalog: {catalog.name}", exc_info=e)
```

Comment:
logger.warning(f"Error while listing schemas in catalog: {catalog.name}", exc_info=e)